app/helpers/api-util.js

```diff
@@ -167,26 +167,27 @@ export async function getByIdPassport(id) {
 
 // PROVERBS
 
-export async function getAllProverbs() {
+export async function getProverbs() {
   try {
     const client = await clientPromise;
     const db = client.db("roads");
 
-    const allProverbs = await db.collection("proverbs").find({}).toArray();
+    const currentDate = new Date();
+    let currentMonth = currentDate.getMonth() + 1;
 
-    return allProverbs.map((mongoDbItem) => {
+    const proverbs = await db
+      .collection("proverbs")
+      .find({ id: currentMonth })
```

Comment:
id implies this is something unique but we are using currentMonth which does not seem like something unique

the proverb id should be unique

the proverb document can have a field called month that you use to filter

Reply:
There is not two months that are the same, that's why I used the id to identify the month

Reply:
but maybe there will be proverbs that have the same month?

id != month

your attribute should be informative about the information they carry

an id implies the value is unique across all proverbs

month would imply the month where the proverb is used

you should also use findOne together with the month attribute

Reply:
Indeed! Changes done, according your suggestions